core/src/test/java/org/apache/gravitino/connector/authorization/TestAuthorization.java

```diff
@@ -73,7 +73,7 @@ public static void setUp() throws Exception {
 
     mySQLCatalog =
         new TestCatalog()
-            .withCatalogConf(ImmutableMap.of(Catalog.AUTHORIZATION_PROVIDER, "mysql"))
+            .withCatalogConf(ImmutableMap.of(Catalog.AUTHORIZATION_PROVIDER, "mysqlTest"))
```

Comment:
use a static variable? since it's shared by different classes.

Reply:
Not other place needs to use this static variable.